editor/editor_help.cpp

```diff
@@ -758,8 +758,9 @@ void EditorHelp::_update_doc() {
 
 			if (cd.theme_properties[i].description != "") {
 				class_desc->push_font(doc_font);
-				class_desc->add_text("  ");
 				class_desc->push_color(comment_color);
+				static const char32_t dash[6] = { ' ', ' ', 0x2013 /* en dash */, ' ', ' ', 0 };
+				class_desc->add_text(String(dash));
```

Comment:
Maybe this could use `String::utf8("  –  ")` instead?

Reply:
I've copied this from enumerations, where it's done like that. If you think it's okay, I can replace it in both places.

Reply:
Or just `class_desc->add_text(U"  –  ");` to avoid any encoding conversion.

Reply:
Done!